scripts/bootstrap

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env bash
+
+#
+# bootstrap
+#
+# Install and configure Opkg for use with Entware and Toltec
+#
+
+bootstrap_wget() {
+    if [ ! -d /home/root/.cache/wget_bin ]
+    then
+        log INFO "Installing a current wget binary with better and current tls support" \
+            "to /home/root/.cache/wget_bin/ for this installation..." \
+            ""
+        # Bootstrap a current version of wget
+        WGET_BINARIES_PATH='http://github.com/LinusCDE/wget-remarkable-pipeline/releases/download/job254'
```

Comment:
For consistency, uppercase should be reserved for environment variables, leaving lowercase to normal variables. By the way, all those variables should probably be marked `local`.

Reply:
It’s still not clear to me why we can’t manually download the [wget from Entware](http://bin.entware.net/armv7sf-k3.2/wget_1.20.3-4_armv7-3.2.ipk) and extract it to get a secure `wget`, instead of relying on another repository.

Reply:
> For consistency, uppercase should be reserved for environment variables, leaving lowercase to normal variables. By the way, all those variables should probably be marked local.

Good find! Should be addressed with commit 35bf74fb.

> It’s still not clear to me why we can’t manually download the wget from Entware and extract it to get a secure wget, instead of relying on another repository.

The "problem" is that entware does a proper job with their packages. They work exclusivly in the /opt directory and don't assume anything about the rest of the system. Even though binaries are dynamically linked, the linker lib itself is always hardcoded in the binary. In case of entwares wget it is on /opt.

Here is an example of me running entwares wget binary without having /opt mounted.
```
reMarkable: ~entwget/ ls
wget      wget-ssl
reMarkable: ~entwget/ ./wget
-sh: ./wget: No such file or directory
reMarkable: ~entwget/ file wget
wget: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), dynamically linked, interpreter /opt/lib/ld-linux.so.3, for GNU/Linux 3.2.0, stripped
reMarkable: ~entwget/ 
```

The `file not found` also confused me first when tinkering with c a while ago. But the interpreter lib `ld-linux.so` is meant with that.

Reply:
Good catch, I didn’t think about that! Then I think that it would be best to move the wget build over to this repo, and to publish it on `https://toltec.delab.re`. I’ll do that in a separate PR if it’s OK with you.

Reply:
No problem. If you want, I can also move the remarkable-wget-pipeline over to the toltec-dev domain. Though a custom gitlab actions build would probably be helpful at that point anyway. So we might as well just recreate that repo. What do you think?

Reply:
A transfer sounds great, I can add a GitHub workflow to it if you want.

Reply:
Is tranferred to toltec-dev. We just to get `buildroot/base:20200814.2228` (from docker hub) to run `ci_build.sh` and zipping the `dist/` folder afterwards.

Reply:
Thanks! I’m not really familiar with buildroot. I wonder if it could be used to simplify the Dockerfiles in toltec-dev/toolchain?

Reply:
> I wonder if it could be used to simplify the Dockerfiles in toltec-dev/toolchain?

Only if you want to build your base image yourself I guess. Buildroot meant to build a Busybox or Systemd bases linux system from scratch along with a bootloader. It uses a similar buildsystem to the Linux kernel and basically downloads and builds every single binary from scratch for many architectures and boards. I use it at work for a RPi-Like board. That way our software will be built into a ready-to-flash sd.img file.

It may be useful for your custom oecore-like toolchain. But I'm not sure whether getting the header files is even a usecase.